site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -506,7 +521,7 @@ The maximum number of digits to use when building a unique suffix for a username
 [field]#{base_field_name}.usernameConfiguration.unique.separator# [type]#[String]# [since]#Available since 1.27.0#::
 A single character to use as a separator from the requested username and a unique suffix that is added when a duplicate username is detected. This value can be a single non alphanumeric ASCII character.
 
-[field]#tenant.usernameConfiguration.unique.strategy# [type]#[String]# [since]#Available since 1.29.0#::
```

Comment:
Thanks for changing these!

Reply:
np.  I thought I'd sneak them in as I saw them instead of opening a new PR.